modules/assim.sequential/R/sda.enkf_MultiSite.R

```diff
@@ -40,6 +40,14 @@ sda.enkf.multisite <- function(settings,
                                             Profiling = FALSE,
                                             OutlierDetection=FALSE),
                                ...) {
+  #add if/else for when restart points to folder instead if T/F set restart as T
+  if(class(restart) == "list"){
```

Comment:
Classes can have more than one name (e.g. try `class(CO2)`) , so comparing them with `==` can fail in unexpected ways. My usual pattern is `if(inherits(obj, "class"))`, but for this case there's an even simpler built-in:
```suggestion
  if(is.list(restart)){
```